vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -251,14 +261,23 @@ func verifyAddSubnetValidatorTx(
 	}
 
 	// Verify the flowcheck
+	var (
+		staticFeesCfg = backend.Config.StaticConfig
+		upgrades      = backend.Config.Times
+	)
```

Comment:
nit: Can we avoid these vars and just in-line them into `New...(`? We're only using them once so I feel like this is overly verbose. Applicable to rest of the files in this package.

Reply:
dropped them all